src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -23,6 +23,16 @@ const { yellow } = uiColors;
 
 const isNullish = (val: any) => val === null || val === undefined;
 
+/**
+ * "Invisible" errors are errors that we want to affect formState (e.g. preventing submission) but
+ * not show visibly on the UI. This function filters out invisible errors so that they do not affect
+ * the visual appearance of the form elements.
+ * @param errors - array of error messages
+ * @returns error messages array with "invisible" errors removed
+ */
```

Comment:
Maybe worth noting here that this bug occurs specifically when using a `validate` function just so that fact is noted somewhere in code.